lib/cartopy/mpl/geoaxes.py

```diff
@@ -1112,12 +1112,12 @@ def imshow(self, img, *args, **kwargs):
             result = matplotlib.axes.Axes.imshow(self, img, *args,
                                                  extent=extent, **kwargs)
 
-        # clip the image. This does not work as the patch moves with mouse
-        # movement, but the clip path doesn't
-        # This could definitely be fixed in matplotlib
-#        if result.get_clip_path() in [None, self.patch]:
-#            # image does not already have clipping set, clip to axes patch
-#            result.set_clip_path(self.outline_patch)
+            # clip the image. This does not work as the patch moves with mouse
+            # movement, but the clip path doesn't
+            # This could definitely be fixed in matplotlib
+        #        if result.get_clip_path() in [None, self.patch]:
```

Comment:
I have to say, this rendering of the diff suggests that these comment characters still aren't indented correctly...

Reply:
I also didn't touch that... what's happening?